number_theory/numeric_unittest.cpp

```diff
@@ -41,6 +41,11 @@ TEST(NumericTest, ExGCD) {
     int64_t b = rand_gen(rand_engine);
     test_exgcd(a, b);
   }
+  // test overflow
+  test_exgcd<int32_t>(std::numeric_limits<int32_t>::min(),
```

Comment:
Is there a need for explicitly testing 32-bit and 64-bit overflow? If not,  we can just use `int` as suggested by Google's style guide such as:
```cpp
test_exgcd(std::numeric_limits<int>::min(),
           std::numeric_limits<int>::max());
```

Reply:
On some 64-bit systems, int is a 32-bit integer type. Since the system uses 64-bit registers, adding two large 32-bit integer may not overflow. So we test both 32-bit and 64-bit integers explicitly.